source/fundamentals/crud/read-operations/change-streams.txt

```diff
@@ -288,3 +288,56 @@ output:
 
 For a list of options, see the `FullDocument <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/changestream/FullDocument.html>`__
 API documentation.
+
+Split Large Change Stream Events
+--------------------------------
+
+Starting in MongoDB 7.0, you can use a ``$changeStreamSplitLargeEvent``
+aggregation stage to split events that exceed 16 MB into smaller fragments.
+``$changeStreamSplitLargeEvent`` returns the fragments sequentially using the
+change stream cursor. Each fragment includes a ``SplitEvent`` object containing
+the following fields:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 35 65
+
+   * - Field
+     - Description
+
+   * - ``fragment``
+     - The index of the fragment, starting at 1
+
+   * - ``of``
+     - The total number of fragments in the event
```

Comment:
Suggestion:

Partially because Vale picked this up, but since "in the event" could be interpreted as an idiom, I think it could be good to reword it, and provide additional detail. E.g.

"The total number of fragments that compose the split change event